### ISSUE TITLE
project name change requests

### ISSUE THREAD

**Issue by quepol** [18F/tock#164]
I'd like to request the following name changes, which will make these projects more easily findable/searchable with the new search functionality.

- Mngr/Supv Activities —> Manager / Supervisor Activities
- 18F-USDS Explore —> 18F - USDS Explore
- API-Program —> API Program
- Currency Consulting —> Consulting / Currency _(for consistency's sake)_
- IT Issues (GSA OIT/Helpdesk) —> IT Issues (GSA OIT / Helpdesk)
- Ops/Acquisitions —> Ops / Acquisitions
- Ops/Admin —> Ops / Admin
- Ops/Budget & Finance —> Ops / Budget & Finance
- Ops/HR —> Ops / HR
- Ops/Legal —> Ops / Legal
- Ops/Project Intake —> Ops / Project Intake
- Ops/Project Luca —> Ops / Project Luca
- Outreach/18F Site —> Outreach / 18F Site
- Outreach/Communications —> Outreach / Communications
- Outreach/Events —> Outreach / Events
- Recruiting/Interviewing —> Recruiting / Interviewing
- SecureHighCloud —> Secure High Cloud
- StyleLibrary —> XD - Style Library _(for consistency)_
- SecurityPackageAutomationTool —> Security Package Automation Tool
- Consulting - Agile BPA —> Consulting / Agile BPA 

Basically anything that has a dash or a slash in it needs to have spaces to make those more easily searchable. And let's get away from CamelCase for names in general.

**Comment by batemapf** [comment]
i think i got all that is listed above, as well as couple others.  take a look @quepol and let me know if i missed any.  if so, please add them in a comment.  if not, please close!

**Comment by quepol** [comment]
- Ops/HR
- Ops/Project Intake
- Recruiting/Interviewing
- StyleLibrary

^ still outstanding according to https://tock.18f.gov/projects/

Thank you!

**Comment by batemapf** [comment]
my end of day eyes missed those.  should be good now.  close or comment as needed.

**Comment by quepol** [comment]
:thumbsup: Thx!!
